Core/src/main/java/com/paypal/android/core/graphql/fundingEligibility/FundingEligibilityQuery.kt

```diff
@@ -0,0 +1,61 @@
+package com.paypal.android.core.graphql.fundingEligibility
+
+import com.paypal.android.core.graphql.common.Query
+import com.paypal.android.core.graphql.fundingEligibility.models.FundingEligibilityIntent
+import com.paypal.android.core.graphql.fundingEligibility.models.FundingEligibilityResponse
+import com.paypal.android.core.graphql.fundingEligibility.models.SupportedCountryCurrencyType
+import com.paypal.android.core.graphql.fundingEligibility.models.SupportedPaymentMethodsType
+import org.json.JSONObject
+
+class FundingEligibilityQuery(
```

Comment:
this could be internal
```suggestion
internal class FundingEligibilityQuery(
```